2-multitenant/terraform.example.tfvars

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
Thank you very much for adding this file! It will make it so much easier to jumpstart with this example :)